R/httpgd.R

```diff
@@ -234,29 +227,25 @@ hgd_renderers <- function(which = dev.cur()) {
 #' @param state Include the current device state in the returned result
 #'  (see also: [hgd_state()]).
 #'
-#' @return TODO
+#' @return a list contains plot IDs.
 #'
 #' @importFrom grDevices dev.cur
 #' @export
 #'
 #' @examples
-#' \dontrun{
-#'
 #' hgd()
+#'
 #' plot.new()
 #' text(.5, .5, "#1")
 #' plot.new()
 #' text(.5, .5, "#2")
 #' plot.new()
 #' text(.5, .5, "#3")
-#' third <- hgd_id()
-#' second <- hgd_id(2)
-#' all <- hgd_id(1, limit = Inf)
-#' hgd_remove(1)
-#' hgd_svg(second)
+#' hgd_id() # The last one
+#' hgd_id(2) # The second one
+#' hgd_id(1, limit = Inf) # The first one and all the followings
```

Comment:
I'm not sure if this is the right example for `hgd_id`, and I'm not sure if it's user friendly since pkgdown shows images.

![image](https://user-images.githubusercontent.com/50911393/214597589-1e23bee8-401e-44a6-be25-634539c6d573.png)

Reply:
Hm, pkgdown showing images is indeed wrong. Does this only happen here? If you have a better idea for an example feel free to edit it. This API will change a bit in httpgd 2.0 / unigd

Reply:
The same thing is happening in all the places where I run plot functions, not just here. But I couldn't figure out how to fix it...
Since you probably don't plan to build the pkgdown site from this branch (for now), why not leave it as is first?

Since even the `plot` function is executing examples, it seems better not to use `if (interactive())` as much as possible.

I will ask at the pkgdown repository about how to fix it on the pkgdown site.

Reply:
Yes we can leave it for now, but we will have the same issue with unigd later so thank you for asking at the pkgdown repo!